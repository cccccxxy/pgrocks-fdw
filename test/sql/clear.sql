--
-- Delete the extension and server 
--

DROP SERVER kv_server;  
DROP EXTENSION kv_fdw;  
  
\c postgres  
DROP DATABASE kvtest;  
