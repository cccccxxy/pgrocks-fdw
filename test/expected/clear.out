--
-- Delete the extension and server
--
DROP SERVER kv_server;
DROP SERVER
DROP EXTENSION kv_fdw;
DROP EXTENSION

\c postgres
You are now connected to database "postgres" as user "postgres".
DROP DATABASE kvtest;
DROP DATABASE
